systemvm/debian/opt/cloud/bin/setup/common.sh

```diff
@@ -583,14 +583,14 @@ setup_ntp() {
 }
 
 routing_svcs() {
-   echo "haproxy apache2" > /var/cache/cloud/enabled_svcs
-   echo "cloud nfs-common portmap" > /var/cache/cloud/disabled_svcs
+   echo "cloud haproxy apache2" > /var/cache/cloud/enabled_svcs
```

Comment:
On VRs we don't need to run any agent that is run by the cloud service.

Reply:
yes we could, i kept the separation in case it was useful in future, (this is why i left the virtualbox case in despite it not required at this time).
I can combine the xen cases if that is preferred.